docs/reference/tags/experience_date_search_tag/index.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: default
+title: Experience date search
+parent: Tags
+grand_parent: Reference
+---
+
+# Experience date search
+
+For additional examples see [here]({% link docs/experience_date_search/index.md %}).
+
+The experience date search tag executes a search on the company's product dates and paginates the results.
+
+##### input
+{% raw %}
+```liquid
+{% experience_date_search name: 'Beyond', duration: { greater_than: 3, less_than: 8 }, page_size: month, sort: 'departure_date_asc' %}
+    {% for item in result.items %}
+        <p>{{ item.name }}</p>
+    {% endfor %}
+    <div>
+        {{ paginate.collection_size }} Results
+        {{ paginate | default_pagination }}
+    </div>
+{% endexperience_date_search %}
+```
+{% endraw %}
+
+##### output
+{% raw %}
+```html
+<p>Beyond Sahara</p>
+<p>Beyond Archipelago</p>
+<p>Beyond Sahara Friends & Family</p>
+<p>Beyond Archipelago Friends & Family</p>
+<p>Beyond Asia</p>
+<div>
+    112 Results
+    <span class="prev">
+    <a href="http://beyondadventures.com/pagination-test?page=2">Previous</a>
+    </span>
+    <span class="page">
+    <a href="http://beyondadventures.com/pagination-test?page=1">1</a>
+    </span>
+    <span class="page">
+    <a href="http://beyondadventures.com/pagination-test?page=2">2</a>
+    </span>
+    <span class="page current">3</span>
+    <span class="page">
+    <a href="http://beyondadventures.com/pagination-test?page=4">4</a>
+    </span>
+    <span class="page">
+    <a href="http://beyondadventures.com/pagination-test?page=5">5</a>
+    </span>
+    <span class="deco">&hellip;</span>
+    <span class="page">
+    <a href="http://beyondadventures.com/pagination-test?page=9">9</a>
+    </span>
+    <span class="next">
+    <a href="http://beyondadventures.com/pagination-test?page=4">Next</a>
+    </span>
+</div>
+```
+{% endraw %}
+
+The tag returns an `items` array and a `Paginate` pagination object to the block passed in, the items array represents 1 page of the results from the search.
+
+
+## Pagination
+
+The results of this search are paginated for performance reasons.
+You can paginate results into equal page sizes, defining the number of results to display per page using the `page_size` attribute. There is a maximum page size of 100.
+Alternatively, you can paginate results by month, by setting the `page_size` to `month`.
+
+## Search Params
+
+Search params can be handled in one of two ways:
+
+### Passing explicit attributes
+
+It's possible to use any of the following attributes in the search tag itself, this is useful for building a page responsible for a specific type of product.
+
+#### active_promotion
+Passing `true` to this will mean all items returned will have a currently active promotion.
+
+#### category
+Will search for products which match the category passed, this must use one of Easol's predefined categories and will accept either a single Category or an array.
```

Comment:
a few instances in this file of 'search for products' which should be 'search for experience dates'?

Reply:
Goo catch. Have scanned the document for more cases similar to this one!